libs/clients/taktikal/src/lib/taktikal.module.ts

```diff
@@ -0,0 +1,40 @@
+import { createEnhancedFetch } from '@island.is/clients/middlewares'
+import { DynamicModule } from '@nestjs/common'
+import { Base64 } from 'js-base64'
+
+import { TaktikalService } from './taktikal.service'
+import { RegisterConfig } from './types/config'
+import { Configuration, FlowApi, SigningApi } from '../../gen/fetch'
+
+export class TaktikalModule {
+  static register(config: RegisterConfig): DynamicModule {
```

Comment:
Using register methods is deprecated. You should instead use the [Config module](https://docs.devland.is/libs/nest/config). Feel free to ping me on slack if you need any help using the Config module.